### ISSUE TITLE
[a11y] hotkey widget should have correct semantics

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161689]
When using https://api.flutter.dev/flutter/widgets/Shortcuts-class.html, the loading spinner should have following role
| OS | role |
|--------|--------|
| web |- |
| ios | - |
| macos | - |
| windows | ROLE_SYSTEM_HOTKEYFIELD |
| android | - |
